components/nds-alphabet/src/Alphabet.js

```diff
@@ -0,0 +1,103 @@
+import React, { Children, isValidElement, cloneElement } from "react";
+import PropTypes from "prop-types";
+import "../scss/alphabet.scss";
+
+export const Alphabet = props => {
+	const { children, id, chunky, ...attrs } = props;
```

Comment:
I think it would be good to pull out `className` as a prop here so it can be appended to the alphabet/alphabet--chunky classes to avoid being overwritten which is also how we do it other places

Reply:
Thanks - adding this